%YAML:1.0
pose_graph_path: "/home/peng/catkin_ws/output/pose_graph/"

baseline: 0.110073808127187
# focal_length: 457.975
image_width: 752
image_height: 480

CX: 367.215
CY: 248.375

# stereo baseline times fx
focal_length: 435.2046959714599

Camera.bf: 47.90639384423901

Camera.fx: 435.2046959714599
Camera.fy: 435.2046959714599
Camera.cx: 367.4517211914062
Camera.cy: 252.2008514404297


cam0_intrinsics: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data: [458.654, 0, 367.215,
          0, 457.296, 248.375,
          0, 0, 1]

cam0_distortion_coeffs: !!opencv-matrix
  rows: 4
  cols: 1
  dt: d
  data: [-0.28340811, 0.07395907, 0.00019359, 1.76187114e-05 ]

cam1_intrinsics: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data: [457.587, 0, 379.999,
          0, 456.134, 255.238,
          0, 0, 1]

cam1_distortion_coeffs: !!opencv-matrix
  rows: 4
  cols: 1
  dt: d
  data: [-0.28368365, 0.07451284, -0.00010473, -3.55590700e-05 ]
  
t_clc2: !!opencv-matrix
  rows: 3
  cols: 1
  dt: d  
  data: [-0.110073808127187, 0.000399121547014141, -0.000853702503358044 ]

R_clc2: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d  
  data: [0.999997256477881, 0.00231206719242389, 0.00037600810241559,
         -0.00231713572328124, 0.999898048506644, 0.0140898358466482,
         -0.000343393120524168, -0.0140906684527146, 0.999900662637728]

# body_T_cam0: !!opencv-matrix
T_i_c0: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [0.0148655429818, -0.999880929698, 0.00414029679422, -0.0216401454975,
           0.999557249008, 0.0149672133247, 0.025715529948,  -0.064676986768,
           -0.0257744366974, 0.00375618835797, 0.999660727178, 0.00981073058949,
           0, 0, 0, 1]

# body_T_cam1: !!opencv-matrix
T_i_c1: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [0.0125552670891, -0.999755099723, 0.0182237714554, -0.0198435579556,
           0.999598781151, 0.0130119051815, 0.0251588363115, 0.0453689425024,
          -0.0253898008918, 0.0179005838253, 0.999517347078, 0.00786212447038,
          0, 0, 0, 1]
